extra-games/zmusic/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=zmusic
+PKGSEC=libs
+PKGDES="GZDoom's music system as a standalone library"
```

Comment:
What? This is a music system, for GZDoom?

Reply:
Yes ... They make the ZMusic a standalone library. Although I found a binary of ZMusic in GZDoom source code package, I have no idea if I should use it.